requirements.txt

```diff
@@ -19,4 +19,5 @@ alembic==0.9.9
 ratelimit==1.4.1
 lxml==4.2.1
 feedparser==5.2.1
+flake8==3.5.0
```

Comment:
Do we need to introduce a requirements_dev ?

Reply:
Maybe, but you added that to requirements originally :smile: Do you want to add a new commit to this P.R. to fix or leave this for now?

Reply:
The flake part discussion should probably be involved in #112